packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -209,24 +211,8 @@ function getTextualContent(content) {
 	);
 }
 
-/**
```

Comment:
why?

Reply:
It doesnt represent the function jsDoc, I moved all of that in the propTypes

Reply:
I have just check we don t have any code completion in JSX based on jsdoc, so seems useless